Dockerfile

```diff
@@ -1,18 +1,38 @@
-FROM quay.io/3scale/ruby:2.1
-MAINTAINER Oriol Martí <oriol@3scale.net> # 2015-06-26
+FROM centos:7 
+MAINTAINER Daniel Cesario <dcesario@redhat.com>
 
-RUN apt-install -y libssl-dev
+USER root
+
+RUN yum -y update \
+	&& yum install -y centos-release-scl \
+	&& yum install -y rh-ruby23 rh-ruby23-ruby-devel \
```

Comment:
rh-ruby23-* can be exposed as an ENV var so we can run with different versions..